src/org/parosproxy/paros/extension/history/ExtensionHistory.java

```diff
@@ -891,6 +891,10 @@ public boolean supportsDb(String type) {
     	return true;
     }
 
+	public HistoryReferencesTable getLogPanelHistoryReferenceTable() {

```

Comment:
- Worth adding the since (JavaDoc) tag.
 - Could be named just `getHistoryReferencesTable` (the `LogPanel` is not important, that's an implementation detail).